include/date/tz.h

```diff
@@ -1196,6 +1196,7 @@ struct tzdb
 #endif
     const time_zone* current_zone() const;
 };
+bool native_to_standard_timezone_name(const std::string& native_tz_name, std::string& standard_tz_name);
```

Comment:
This is an exposure of a previously static function to convert between windows and IANA timezones. Its previously been windows only but we'll want access to it for Transport Networks.